etc/generate-notices.pl

```diff
@@ -0,0 +1,438 @@
+#!/usr/bin/env perl
+use v5.10;
+use strict;
+use warnings;
+use utf8;
+use open qw/:std :utf8/;
+use File::Basename qw/basename/;
+use File::Find qw/find/;
+
+my @license_files;
+
+find(
+    sub {
+        return unless /^licen[cs]e(?:\..+)?$/i || /\.licen[cs]e$/i;
+        push @license_files, [ $File::Find::dir, $File::Find::name ];
+    },
+    'vendor'
+);
+
+print forked_licenses();
+
+for my $entry (sort { $a->[0] cmp $b->[0] or $a->[1] cmp $b->[1] } @license_files) {
+    ( my $package_name = $entry->[0] ) =~ s{vendor/}{};
+    my $file = basename($entry->[1]);
+    my $license_text = do { local ( @ARGV, $/ ) = $entry->[1]; <> };
+    $license_text =~ s/ +$//mg;
+    say "" x 70;
```

Comment:
(I know you didn't write this code, but lol at `say "" x 70`, to be really sure you printed nothing.)

Reply:
Heh, I never noticed this.